class_configs/brd_class_config.lua

```diff
@@ -740,73 +752,304 @@ local _ClassConfig = {
+                cond = function(self, aaName)
+                    return RGMercUtils.AAReady(aaName)
+                end,
+            },
+            {
+                name = "Dance of Blades",
+                type = "AA",
+                cond = function(self, aaName)
+                    return RGMercUtils.AAReady(aaName)
+                end,
+            },
+            {
+                name = "Cacophony",
+                type = "AA",
+                cond = function(self, aaName)
+                    return RGMercUtils.AAReady(aaName)
+                end,
+            },
+            {
+                name = "Frenzied Kicks",
+                type = "AA",
+                cond = function(self, aaName)
+                    return RGMercUtils.AAReady(aaName)
+                end,
+            },
         },
         ['Debuff'] = {
+            {
+                name = "MezAESong",
+                type = "Song",
+                cond = function(self, songSpell)
+                    local setting = RGMercUtils.GetSetting('DoAEMez')
+                    local memmed = RGMercUtils.SongMemed(songSpell)
+                    local spellReady = mq.TLO.Me.SpellReady(songSpell)()
+                    local aeMezCt = RGMercUtils.GetSetting("AEMezCount")
+
+                    local res = setting and memmed and spellReady
+                        and RGMercUtils.GetXTHaterCount() >= aeMezCt
+
+                    return res
+                end,
+            },
+            {
+                name = "AESlowSong",
+                type = "Song",
+                cond = function(self, songSpell)
+                    return RGMercUtils.GetSetting("DoAESlow")
+                        and RGMercUtils.DetSpellCheck(songSpell)
+                        and RGMercUtils.GetXTHaterCount() > 2
+                        and not mq.TLO.Target.Slowed()
+                end,
+            },
+            {
+                name = "SlowSong",
+                type = "Song",
+                cond = function(self, songSpell)
+                    return RGMercUtils.GetSetting("DoSlow")
+                        and RGMercUtils.DetSpellCheck(songSpell)
+                        and not mq.TLO.Target.Slowed()
+                end,
+            },
+            {
+                name = "DispelSong",
+                type = "Song",
+                cond = function(self, songSpell)
+                    return RGMercUtils.GetSetting('DoDispel') and mq.TLO.Target.Beneficial()
+                end,
+            },
+            {
+                name = "Dreadstone",
+                type = "Item",
+                cond = function(self, itemName)
+                    -- This item is instant cast for free with almost no CD, just mash it forever when it's available
+                    return mq.TLO.FindItem(itemName).TimerReady() == 0
+                end,
+            },
         },
         ['Heal'] = {
         },
-        ['DPS'] = {
+
+        ['Testing'] = {
```

Comment:
Should this rotation go to prod?

Reply:
Nope, same case as the above Procedural one.  Clearly I should stop folding away my code